WooCommerce/Classes/ViewRelated/Products/Add Product/AddProductWithAI/StartingInfo/ProductCreationAIStartingInfoView.swift

```diff
@@ -0,0 +1,177 @@
+                        .foregroundColor(Color(.brand))
+                        .bodyStyle()
+                }
+            }
+            Spacer()
+        }
+    }
+
+    var placeholderText: some View {
+        Text(Localization.placeholder)
+            .foregroundColor(Color(.placeholderText))
+            .bodyStyle()
+            .padding(insets: Layout.placeholderInsets)
+            // Allows gestures to pass through to the `TextEditor`.
+            .allowsHitTesting(false)
+            .renderedIf(viewModel.features.isEmpty)
+    }
+
+    var continueButton: some View {
+        Button {
+            // continue
+            editorIsFocused = false
+            viewModel.didTapContinue()
+            onContinueWithFeatures(viewModel.features)
+        } label: {
+            Text(Localization.continueText)
+        }
+        .buttonStyle(PrimaryButtonStyle())
+        .disabled(viewModel.features.isEmpty)
+    }
+}
+private extension ProductCreationAIStartingInfoView {
+    enum Layout {
+        static let insets: EdgeInsets = .init(top: 24, leading: 16, bottom: 16, trailing: 16)
+
+        static let titleBlockBottomPadding: CGFloat = 40
+
+        static let titleBlockSpacing: CGFloat = 16
+
+        static let editorBlockSpacing: CGFloat = 8
+        static let minimumEditorHeight: CGFloat = 70
+        static let cornerRadius: CGFloat = 8
+        static let messageContentInsets: EdgeInsets = .init(top: 10, leading: 10, bottom: 10, trailing: 10)
+        static let placeholderInsets: EdgeInsets = .init(top: 16, leading: 16, bottom: 16, trailing: 16)
+        static let dividerHeight: CGFloat = 1
+        static let readTextFromPhotoButtonInsets: EdgeInsets = .init(top: 11, leading: 16, bottom: 11, trailing: 16)
+
+        enum UsePackagePhoto {
+            static let padding: CGFloat = 4
+            static let spacing: CGFloat = 4
+            static let cameraSFSymbol = "camera"
+        }
+    }
+
+    enum Localization {
+        static let title = NSLocalizedString(
+            "Starting information",
+            comment: "Title on the starting info screen."
+        )
+        static let subtitle = NSLocalizedString(
+            "Tell us about your product, what it is and what makes it unique, then let the AI work its magic.",
+            comment: "Subtitle on the starting info screen."
+        )
+        static let placeholder = NSLocalizedString(
+            "For example: Black cotton t-shirt, soft fabric, durable stitching, unique design",
+            comment: "Placeholder text on the product features field"
+        )
+        static let readTextFromPhoto = NSLocalizedString(
+            "Read text from product photo",
+            comment: "Upload package photo button on the text field"
+        )
+        static let continueText = NSLocalizedString(
+            "Continue",
+            comment: "Continue button on the starting info screen."
+        )
+    }
```

Comment:
Should we follow the latest naming style for localized strings with separate keys and values?

Reply:
Good point. Done in https://github.com/woocommerce/woocommerce-ios/pull/13171/commits/1307d5d13f94c346dcf2778e8c3589ef08750b08